Controller/Payment/Notify.php

```diff
@@ -140,6 +148,13 @@ public function execute()
                 $kevinSignature = $this->getRequest()->getHeader('X-Kevin-Signature');
```

Comment:
Is `getHeader()` case insensitive? However, please use our `SecurityManager` inside PHP lib. Check signature as soon as possible.

https://github.com/saltanovas/kevin-php#41-verify-signature

Reply:
It's not casesensitive, but I will use SecurityManager